BcSaaS/Import-BcConfigurationPackage.ps1

```diff
@@ -0,0 +1,161 @@
+﻿<# 
+ .Synopsis

```

Comment:
Please move to the SaaS folder instead of BcSaaS (I might have renamed the folder)
Also please rename to Import-ConfigPackageToBcEnvironment